src/app/ui/NewTaskForm.tsx

```diff
@@ -81,17 +89,24 @@ export const NewTaskForm = () => {
           </Stack>
         </Stack>
       </AppMargin>
-      <NewTaskFooter />
+      <NewTaskFooter handleCreate={handleCreate} />
     </NewTaskContainer>
   )
 }
 
 const NewTaskFormInputs = () => {
+  const { title, description } = useSelector(selectCreateTask)
+
   return (
     <>
       <Stack direction="column" rowGap={1}>
         <Typography variant="md">Task name</Typography>
-        <StyledTextField type="text" padding="8px 0px" />
+        <StyledTextField
+          type="text"
+          padding="8px 0px"
+          value={title}
+          onChange={(e) => store.dispatch(setCreateTaskFields({ targetField: 'title', value: e.target.value }))}
+        />
```

Comment:
It could be a good idea to move the whole `store.dispatch(setCreateTaskFields({ targetField: 'INSERT FIELD HERE', value: e.target.value }))}` to a reusable function. I see this logic rewritten all over the place so.

Reply:
I think `setCreateTaskFields` itself is a reusable reducer imported from the redux slice. IMO creating another abstraction on top of this is not necessary. Having said this, please let me know your thoughts further.